cmd/sentinel/cmd/sentinel_test.go

```diff
@@ -349,9 +349,9 @@ func TestUpdateCluster(t *testing.T) {
 						Generation: 1,
```

Comment:
Is it possible to remove the formatting for the parts that we did not modify? This is going to cause a lot of pain when we try to upstream, or rebase on the upstream repo.

Reply:
I think it should be fine, and you can use diff -w to ignore the white space changes. 

Even though it unnecessarily makes a massive diff to upstream, it is now correctly go fmt'ed.